src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ForwardCancellationTokenToInvocationsTests.cs

```diff
@@ -3257,48 +3191,6 @@ End Class
             return VerifyVB.VerifyCodeFixAsync(originalCode, fixedCode);
         }
 
-        [Fact]
-        public Task VB_Diagnostic_OtherClassStaticMethod_TokenDefault_WithConfigureAwait()
```

Comment:
Duplicates of `VB_Diagnostic_OtherClass_TokenDefault_WithConfigureAwait`.

Reply:
Instead of deleting, let's update the test and make it `Shared Sub` to match the name

Reply:
+1